CONTRIBUTING.md

```diff
@@ -6,3 +6,6 @@ Agreement](https://cla-assistant.io/redfin/react-server). The purpose
 of this license is to protect contributors, Redfin, as well as users
 of this software project. Signing this agreement does not affect your
 rights to use your contributions for any other purpose.
+
```

Comment:
@davidalber lemme know when you've completed whatever offline review you feel is necessary.